CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ## Upgrade Notes
 
+- **Bump Ruby to v2.6**
```

Comment:
I'm seeing on the git history and I think it's the first time that we're communicating an upgrade of the Ruby version. I think we should remove this. 

Also, please let's review again the consensus on #5908 - I think the only relevant section on the CHANGELOG would be the "**Stable branches nomenclature changes**" - the other ones the idea was to regenerate it when the 0.23.0 release is done.

Reply:
I think that it is interesting for implementors to known what they have to take into account when upgrading from one version to the next. But anyway the changelog entry for the Ruby upgrade will appear in the changelog when releasing, although not so clear for implementors.

Reply:
I saw also it was not the first time: https://github.com/decidim/decidim/pull/4927/files

Reply:
Ok, but please let's try to keep the notices for developers on the CHANGELOG short and relevant.